app.js

```diff
@@ -1,131 +1,157 @@
-  
-const inputTeskt = document.querySelector('.input__tekst');
-const inputSetTime = document.querySelector('.time');
-const btnAdd = document.querySelector('.btnPlus');
-const ulList = document.querySelector('.ul-list');
-const alertInfo = document.querySelector('.alert-info')
+  const inputTeskt = document.querySelector('.input__tekst');
+  const inputSetTime = document.querySelector('.time');
+  const btnAdd = document.querySelector('.btnPlus');
+  const ulList = document.querySelector('.ul-list');
+  const alertInfo = document.querySelector('.alert-info')
+
+  let img = null;
+  let timeFromInput = null;
 
 
+  // na ostatnich zajeciach pokazywałes żeby sobie tworzyć coś takiego jak eventList, ale nie wiem jak to zastosować żeby działało, może dałoby sie to pokazać jeszcze raz ale na całym przykładzie ktory będzie to realizował , jeśli to nie problem 
+  // pokazałeś coś takiego poniżej : hmm wiem że to jest tablica z 3 obiektami 
 
-let img = null;
+  //   const eventList = [{
+  //           time: 500,
+  //           url: 'http://google.com/image.png',
+  //           loaded: true
+  //       },
+  //       {
+  //           time: 400,
+  //           url: 'http://google.com/image.png',
+  //           loaded: true
+  //       },
+  //       {
+  //           time: 300,
+  //           url: 'http://google.com/image.png',
+  //           loaded: true
+  //       },
+  //   ]
 
```

Comment:
jasne - na początek nie masz żadnych eventów, więc po prostu tworzysz sobie pustą listę
const eventList = []

potem w momencie kiedy Tworzysz kolejny wpis to robisz tylko 
`eventList.push({ time: <iles czasu>, url: <zawartosc z inputa >})
renderElements(eventList)`

Gdzie render Elements tworzy po prostu w jakims kontenerze to co ma się wyświetlić. (wczesniej ofc czyszczac zawartosc)

Jesli chcesz coś usunąć to też po prostu szukasz po tablicy - usuwasz, i znowu renderElements na wyczyszczonej tablicy itd itd.

W momencie dodania, możesz dodać sobie setTimeout ktory w momencie uplyniecia czasu odpali promise i dajesz loaded na true i znowu.. renderElements ktory wie ze jak jest ten klucz to wyswietli juz obrazek :)

Reply:
dziękuję za wyjaśnienie